arroyo-console/src/udf_state.ts

```diff
@@ -135,9 +135,12 @@ export const getLocalUdfsContextValue = () => {
     const id = generate_udf_id();
     const functionName = `new_udf`;
     const definition =
+      `/*\n` +
+      `[dependencies]\n\n` +
```

Comment:
I actually did remove the default dependencies with this change. I personally think this is clear as is, since anyone writing rust code and who knows about a dependency existing should be somewhat familiar with a Cargo.toml. Maybe a better place for that kind of hand holding is through the Catalog thing. Like we can add an example in here with a UDF + dependencies: 
![image](https://github.com/ArroyoSystems/arroyo/assets/8881183/5490b130-2574-454b-86b9-5c412ac5c144)